app/models/export_format.rb

```diff
@@ -35,7 +35,8 @@ class ExportFormat < ActiveRecord::Base
     'payments.id',
     'offsite_payments.check_number',
     'donations.comment',
-    'misc_payment_infos.fee_covered'
+    'misc_payment_infos.fee_covered',
+    'donations.created_at'
```

Comment:
We want this to be JUST for recurring donations I think. I don't see any advantage in showing when a donation was created if it's a one time. What do you think?

Reply:
Updated!